docs/Common-Issues.md

```diff
@@ -113,3 +113,7 @@ Following a macOS upgrade it may be necessary to reinstall the Xcode Command Lin
 xcode-select --install
 brew upgrade
 ```
+
+### Local build failures
+
+Build failures on one machine, but not similarly equipped machines (check issues for other projects too), can indicate that the install is borked. If `brew doctor` doesn’t, reinstalling may help. That is, `brew bundle dump`, [uninstall](https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew), [reinstall](https://docs.brew.sh/Installation), and `brew bundle install`.
```

Comment:
We should probably use something other than "borked" here, as I don't think it's a term that's universally understood.

Emphasising that `brew doctor` often contains instructions to help fix things might be useful. It's still quite surprising how many users think `brew doctor` is something you just run, instead of read and try to implement.